### PR TITLE
一括配送機能

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

File: stock_picking_validate/i18n/ja.po [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#. module: stock_picking_validate
+#: model:ir.ui.view,arch_db:stock_picking_validate.stock_picking_validate_wizard
+msgid "Stock Picking Validate"
+msgstr "一括配送"
+
+#. module: stock_picking_validate
+#: model:ir.ui.view,arch_db:stock_picking_validate.stock_picking_validate_wizard
+msgid "This wizard will allow validate bunch of Stock Picking."
+msgstr "選択した配送レコードを配送検証して移動します。"
```
Comment (TimLai125) [inline stock_picking_validate/i18n/ja.po]: @yostashiro Can you review this translation string

Comment (yostashiro) [reply stock_picking_validate/i18n/ja.po]: My personal preference is "選択したピッキングを一括検証します。"

File: stock_picking_validate/i18n/ja.po [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+#. module: stock_picking_validate
+#: model:ir.model.fields,field_description:stock_picking_validate.field_stock_picking_validate_write_date
+msgid "Last Updated on"
+msgstr "最終更新日"
+
+#. module: stock_picking_validate
+#: model:ir.ui.view,arch_db:stock_picking_validate.stock_picking_validate_wizard
+msgid "Stock Picking Validate"
+msgstr "一括配送"
```
Comment (yostashiro) [inline stock_picking_validate/i18n/ja.po]: ピッキング一括検証

File: stock_picking_validate/i18n/ja.po [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+#. module: stock_picking_validate
+#: model:ir.ui.view,arch_db:stock_picking_validate.stock_picking_validate_wizard
+msgid "This wizard will allow validate bunch of Stock Picking."
+msgstr "選択した配送レコードを配送検証して移動します。"
+
+#. module: stock_picking_validate
+#: model:ir.actions.act_window,name:stock_picking_validate.action_stock_picking_validate_window
+#: model:ir.ui.view,arch_db:stock_picking_validate.stock_picking_validate_wizard
+msgid "Validate"
+msgstr "一括配送"
```
Comment (yostashiro) [inline stock_picking_validate/i18n/ja.po]: 一括検証

File: stock_picking_validate/wizard/stock_picking_validate_wizard.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_picking_validate_wizard" model="ir.ui.view">
+        <field name="name">stock.picking.validate.wizard</field>
+        <field name="model">stock.picking.validate</field>
+        <field name="arch" type="xml">
+            <form string="Stock Picking Validate">
+                <p class="oe_gray">This wizard will allow validate bunch of Stock Picking.</p>
```
Comment (yostashiro) [inline stock_picking_validate/wizard/stock_picking_validate_wizard.xml]: "This wizard will validate selected stock pickings at once. Make sure that you have made no mistake with selection."

File: stock_picking_validate/wizard/stock_picking_validate.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+from odoo import models
+
+
+class StockPickingValidate(models.TransientModel):
+    _name = 'stock.picking.validate'
+
+    def action_stock_picking_validate(self):
+        context = dict(self._context or {})
+        active_ids = context.get('active_ids', [])
+        picking_ids = self.env['stock.picking'].browse(active_ids)
```
Comment (yostashiro) [inline stock_picking_validate/wizard/stock_picking_validate.py]: We may insert a check on picking type - show error if there is more than one picking type involved.  My concern is that someone could make a mistake and validate a bunch of random pickings.  This check will not be perfect, but should reduce the risk a bit.  Can you think of any other reasonable measures?

**Pull request merged**